README.md

```diff
@@ -1,59 +1,5 @@
-  * Work in a branch on your fork
-  * Write all of your code in a directory named `lab-` + `<your name>` **e.g.** `lab-duncan`
-  * Submit a pull request to this repository
-  * Submit a link to your pull request on canvas
-  * Submit a question, observation, and how long you spent on canvas  
-  
-## Learning Objectives  
-* Students will understand how to implement a TCP server using the net module
-* Students will understand work with Node.js EventEmitters
-
-## Resources  
-* [Nodejs net docs](https://nodejs.org/api/net.html)
-
-## Requirements  
-#### Configuration  
-<!-- list of files, configurations, tools, ect that are required -->
-Your lab directory must include  
-* **README.md** -- with a documention about your lab
-* **.gitignore** -- with a robust gitignore
-* **.eslintrc** -- with the class .eslintrc file
-* **.eslintignore** -- with the class .eslintignore
-* **.package.json** -- with all dependencies and dev-dependencies 
-* **lib/**  -- directory for holding program helper modules
-* **server.js** --  main server program
- 
-#### Feature Tasks  
-* Create a **server.js** that start a TCP server using the `net` module
-* create a **lib/parse-message.js** module for parsing client request
- * if a client sends a message that starts with `/nick some_name` 
-   * change their nickname to the name they have provided
- * if a client sends a message that starts with `/dm some_name some message`
-   * send their message to the client with the some_name nickname
- * if a client sends a message that starts with `/users`
-   * send the requester the usernames of all connected users 
- * if a clinet sends a message that starts with `/troll some message`
-   * send their message to all users 10 times
- * if a client sends a message that starts with `/ban user_name`
-   * logout the user with the user_name (close their connection, and remove from sockets array)
- * otherwise send their message to all clients
-* when a user speaks their nickname should be printed
- * **i.e.** `teapot: Sup Hacker?`
-
-####  Documentation  
-* Write a project description in your **README.md**
-* Write docs on how to connect to your server with telnet
-
-#### Testing  
-*  No Testing Today :)
-
-## Rubric  
-* 2ps Configuration
-* 3pts Feature Tasks
-* 3pts Tests (free points today)
-* 2pts Documentation
+## Description
+  Nickname command and dm commands copied over prework from the lecture today. Ban command used a similar protocol to the dm command except "s.write" was replaced with "s.end". Originally, I tried using "s.destroy" but that killed the entire server. Users command initially tried using a method called "server.getconnections" which gave the total number of connections but did not elaborate further. Changed this up by adding a forEach loop and just iterating throughout the entire array to find each individual user. Troll command uses a for loop and just iterates through the same message 10 times to annoy other users.
```

Comment:
Great description of how your project went. It helps a lot to show your thought process and how your attempts to tackle the problem and find solutions. 

Remember to read README instructions carefully. You also need to treat this as instructions for someone on how to use the chat server. E.g. from Point A to Point Z, what should I do to start a chat if I had very little experience with using the terminal and how exactly should I type my commands?